docs/native-modules.md

```diff
@@ -51,7 +51,9 @@ Open the Visual Studio solution in the `windows` folder and add the new files di
 | `ReactConstant`         | Specifies a field or property that represents a constant. |
 | `ReactConstantProvider` | Specifies a method that provides a set of constants.      |
 | `ReactEvent`            | Specifies a field or property that represents an event.   |
-| `ReactStruct`           | Specifies a `struct` that can be used in native methods.    |
+| `ReactStruct`           | Specifies a `struct` that can be used in native methods.  |
+| `ReactInit`             | Specifies a class initialization module.                  |
+| `ReactFunction`         | Specifies a field that helps calling a JavaScript function. |

```

Comment:
Just by looking at this sentence I don't know what `ReactFunction` does :(
In NativeModules.h we have 'REACT_FUNCTION annotates a field that helps calling a JavaScript function.' so basically the same thing.
REACT_METHOD specifies the exported function to JS form native, what does this one do though?

Reply:
I agree, that doesn't really help differentiate. I took that string from a previous comment from @vmoroz. Vladimir, is there a better way to capture the difference between REACT_METHOD and REACT_FUNCTION?